cmd/ipfs/main_test.go

```diff
@@ -1,17 +1,20 @@
 package main
 
 import (
-	"testing"
-
 	"github.com/ipfs/go-ipfs/commands"
+
+	gc "gx/ipfs/QmPp8w6dCzx6vPNre8QVSbN7GCY9M7ornWesnrJg48vYxE/check.v1"
 )
 
-func TestIsCientErr(t *testing.T) {
-	t.Log("Catch both pointers and values")
-	if !isClientError(commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified value")
-	}
-	if !isClientError(&commands.Error{Code: commands.ErrClient}) {
-		t.Errorf("misidentified pointer")
-	}
+func (m mainRun) TestIsCientErr(c *gc.C) {
```

Comment:
You are free to have your opinions, but to convince me that this is worthwhile and not just some random contributor trying to push their opinions on a project they just found you'll have to do better convincing me of this. 

You need to choose better examples to prove your point, because this specific one is not helped by shoving a test lib into it. "expected: true, got: false" Is quite clear from reading the existing test and the debuggability is not helped by adding in the extra cognitive overhead of this framework.